justfile

```diff
@@ -1,11 +1,14 @@
 # To use this file, install Just: cargo install just
 set dotenv-load
+# eports all justfile variables as environment variables
+set export
+
 line_length := "100"
 coordinator_log_file := "$PWD/data/coordinator/regtest.log"
 maker_log_file := "$PWD/data/maker/regtest.log"
 pubspec := "$PWD/mobile/pubspec.yaml"
-testnet_coordinator := "COORDINATOR_P2P_ENDPOINT=038fb0eb9d9deeb4f0aa885d6c64abb0a26002871d2f7501349cd8d00f1f130045@195.201.81.94:9045"
-testnet_electrs := "ELECTRS_ENDPOINT=195.201.81.94:50000"
+TESTNET_COORDINATOR := "038fb0eb9d9deeb4f0aa885d6c64abb0a26002871d2f7501349cd8d00f1f130045@195.201.81.94:9045"
+TESTNET_ELECTRS := "195.201.81.94:50000"
```

Comment:
why are these uppercase? I'd suggest to stick to the same convention and keep them lowercase...

Reply:
that is indeed quite random, I can change that :)